lib/advisor/core/questionnaire/form.ex

```diff
@@ -1,8 +1,8 @@
 defmodule Advisor.Core.Questionnaire.Form do
   alias Advisor.Core.{People, Questions}
 
-  def data() do
-    everybody = People.everybody()
+  def data_of(connected_user) do
```

Comment:
In this specific case yes. The user logged in will be the person requesting advice from other people.